test/e2e/specs/editor/various/pattern-overrides.spec.js

```diff
@@ -472,7 +472,7 @@ test.describe( 'Pattern Overrides', () => {
 		await editor.showBlockToolbar();
 		await page
 			.getByRole( 'toolbar', { name: 'Block tools' } )
-			.getByRole( 'link', { name: 'Edit original' } )
+			.getByRole( 'button', { name: 'Edit original' } )
```

Comment:
This is perhaps a little controversial, as I think a link is generally considered more semantic and provides better info for screenreaders. 🤔

Reply:
Yeh, unfortunately until we get proper routing set up in the post editor we can't use a link here.